annotations/2.0/apidocs/META-INF/MANIFEST.MF

```diff
@@ -1,5 +1,2 @@
 Manifest-Version: 1.0
```

Comment:
We should not be including the META-INF directory in the javadocs.  This comment would apply to all three files specified in this PR.

Reply:
Fixed.